apps/design/frontend/src/encode_metadata.ts

```diff
@@ -0,0 +1,115 @@
+import { assert, iter } from '@votingworks/basics';
+
+type Bit = 0 | 1;
+
+function numberToBitsRtl(num: number, bitLength: number): Bit[] {
+  return num
+    .toString(2)
+    .padStart(bitLength, '0')
+    .split('')
+    .map((char) => (char === '0' ? 0 : 1))
+    .reverse();
+}
```

Comment:
Is it convention to use RTL for the metadata row?

Reply:
All of this metadata stuff is based on the Accuvote encoding that our interpreter knows how to read. I actually wrote all this code a while back when designing ballots manually in Figma and just copied it over.

Soon we'll be replacing this with our own metadata encoding scheme, likely via QR code

Reply:
Got it!